src/main/java/com/developers/sosyalapp/service/AccountService.java

```diff
@@ -65,6 +67,7 @@ public ApiResponse<CreateAccountResponse> createAccount(CreateAccountRequest req
             newAccount.setAccountProperties(accountProperties);
 
             Account account = accountRepository.save(newAccount);
+            logger.info("created account");
```

Comment:
Log mesajlarını daha detaylı yazalım, dinamik değerler ekleyelim. Mesela logger.info("Account {} is created", username);